docs/guidance/rayserve-dev-doc.md

```diff
@@ -0,0 +1,128 @@
+# Develop Ray Serve Python scripts on RayCluster
+
+Many RayService issues are related to the Ray Serve Python scripts.
+We strongly recommend that users develop and test their Ray Serve scripts in a RayCluster using the same Ray image before deploying them to a RayService.
+In this tutorial, we will show you how to develop [ray-service.mobilenet.yaml](https://github.com/ray-project/kuberay/blob/master/ray-operator/config/samples/ray-service.mobilenet.yaml).
```

Comment:
Would be better to add a 1 sentence description of what `mobilenet` is and why we're using it in the tutorial

Reply:
Updated: https://github.com/ray-project/kuberay/pull/1250/commits/638d7bc493aa446013ef11639076fc56d5cd7f91